### PR TITLE
feat: Added support for CRUD operations with Flows and Flow Entries

### PR DESCRIPTION
**Pull request by craigtweedy**
## Status

* ✅ Ready

## Type

* ### Feature
  Implements CRUD operations for Flows and Flow Entries

## Description

Added support for CRUD operations on `/flows` and CRUD operations on `/flows/<slug>/entr[y,ies]`. Requires client credential authentication.

### CODE REVIEW

File: src/endpoints/flows.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  GetEntry(slug, entry) {
+      return this.request.send(`${this.endpoint}/${slug}/entries/${entry}`, 'GET');
+  }
+
+  CreateEntry(slug, body) {
+    return this.request.send(`${this.endpoint}/${slug}/entries`, 'POST', body);
+  }
+
+  UpdateEntry(slug, entry, body) {
```
Comment (notrab) [inline src/endpoints/flows.js]: I'd probably rename `entry` to `entryId`.

File: src/endpoints/flows.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    return this.request.send(`${this.endpoint}/${slug}/entries/${entry}`, 'PUT', body);
+  }
+
+  DeleteEntry(slug, entry) {
```
Comment (notrab) [inline src/endpoints/flows.js]: I'd probably rename `entry` to `entryId`.

**Comment by craigtweedy** [comment]
@notrab Can you review again?

**Comment by notrab** [comment]
Looks good. Will approve.

**Pull request merged**